### PR TITLE
Students Registration form

### PR DESCRIPTION
**Pull request by SulimanJoya**
What I did in this project

1. clone my trial project 
2. created a new repo (Students-Registration-Form)
3. create a new branch (Registration-Form)
4. push the project for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    <meta charset="UTF-8" />
+    <title>Registration Form</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="header">
+        <h1 id="title" class="text-center">Students Registration Form</h1>
+        <h1 id="description" class="description text-center">
```
Comment (SulimanJoya) [inline index.html:13]: please use one h1 in one page for the purpose of search engine optimization

File: index.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Registration Form</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="header">
```
Comment (SulimanJoya) [inline index.html:11]: please use semantic tags like header instead of div for accessibility

**Review by SulimanJoya — COMMENTED** [review]
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there
try again

#Required Changes ♻️
1. pleases document your work properly and add descriptive README file
2. And please use liners for your project to prevent the errors 
3. finally read the below comments and summit when your done.
thanks 😁

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @SulimanJoya,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Clear and polite comments. ✔️
 - Very professional comment. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**